### ISSUE TITLE
FAILED: SqsSinkTest.sendToQueueWithBatches

### ISSUE THREAD

**Issue by 2m** [akka/alpakka#915]
```
[error] Test akka.stream.alpakka.sqs.javadsl.SqsSinkTest.sendToQueueWithBatches failed: java.lang.IllegalStateException: IO Listener actor terminated unexpectedly for remote endpoint [localhost:0], took 0.156 sec
[error]     at akka.stream.impl.io.ConnectionSourceStage$$anon$2.receive(TcpStages.scala:100)
[error]     at akka.stream.impl.io.ConnectionSourceStage$$anon$2.$anonfun$preStart$1(TcpStages.scala:61)
[error]     at akka.stream.impl.io.ConnectionSourceStage$$anon$2.$anonfun$preStart$1$adapted(TcpStages.scala:61)
[error]     at akka.stream.stage.GraphStageLogic$StageActor.internalReceive(GraphStage.scala:230)
[error]     at akka.stream.stage.GraphStageLogic$StageActor.$anonfun$callback$1(GraphStage.scala:198)
[error]     at akka.stream.stage.GraphStageLogic$StageActor.$anonfun$callback$1$adapted(GraphStage.scala:198)
[error]     at akka.stream.impl.fusing.GraphInterpreter.runAsyncInput(GraphInterpreter.scala:454)
[error]     at akka.stream.impl.fusing.GraphInterpreterShell$AsyncInput.execute(ActorGraphInterpreter.scala:465)
[error]     at akka.stream.impl.fusing.GraphInterpreterShell.processEvent(ActorGraphInterpreter.scala:560)
[error]     at akka.stream.impl.fusing.ActorGraphInterpreter.akka$stream$impl$fusing$ActorGraphInterpreter$$processEvent(ActorGraphInterpreter.scala:742)
[error]     at akka.stream.impl.fusing.ActorGraphInterpreter$$anonfun$receive$1.applyOrElse(ActorGraphInterpreter.scala:757)
[error]     at akka.actor.Actor.aroundReceive(Actor.scala:517)
```

https://travis-ci.org/akka/alpakka/jobs/371653950

**Comment by randomcoder** [comment]
This _should_ have been fixed in `akka-http-10.1.x`.

https://github.com/akka/akka-http/issues/1837

Re: #913

**Comment by ennru** [comment]
The implementation has changed with AWS client 2.0.
Closing.
